Misc/NEWS.d/next/Library/2020-09-26-19-45-06.bpo-41867.hcueGL.rst

```diff
@@ -0,0 +1,2 @@
+Include the different options for `timespec` in docstrings of `isoformat`
+methods.
```

Comment:
Please remove this, as a NEWS entry isn't necessary when only making doc-string improvements.

Reply:
Done.